src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Steps/TestContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Services;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Steps
+{
+    public class TestContext : IDisposable
+    {
+        public ApprenticeCommitmentsTestApi Api { get; set; }
+
+        public TestMessageBus TestMessageBus { get; set; }
+
+        public List<IHook> Hooks { get; } = new List<IHook>();
+        internal ApprenticeCommitmentApiTestServer FunctionsServer { get; set; }
+
+        public void Dispose()
+        {
+        }
+    }
```

Comment:
This dispose method should call dispose on Api and TestMessage bus as these are disposable types  and this object owns them